src/client/client.cc

```diff
@@ -1,6 +1,6 @@
-/* 
+/*
    Mathieu Stefani, 29 janvier 2016
-   
+
```

Comment:
Please do not make syntax/formatting changes in a PR that is also designed to change functionality.  Please back out all changes that affect whitespace from this PR.